src/lib/concurrency/transaction_manager.hpp

```diff
@@ -57,6 +57,10 @@ class TransactionManager : private Noncopyable {
    */
   std::shared_ptr<TransactionContext> new_transaction_context();
 
+  // TransactionID = 0 means "not set" in the MVCC columns
```

Comment:
That should get a short explanation in which scenario a MVCC TID can be "not set"